packages/deployments/contracts/contracts_forge/core/connext/helpers/RelayerProxyHub.t.sol

```diff
@@ -0,0 +1,483 @@
+      _sendRoot: bytes32(uint256(2)),
+      _blockHash: bytes32(uint256(3)),
+      _proof: _proof,
+      _index: 69,
+      _message: IArbitrumHubConnector.L2Message({
+        l2Sender: address(0),
+        to: address(1),
+        l2Block: 24,
+        l1Block: 48,
+        l2Timestamp: 256,
+        value: 10,
+        callData: abi.encode("data")
+      })
+    });
+    proxy.processFromRootKeep3r(abi.encode(params), 42161, bytes32(uint256(4)));
+
+    vm.mockCall(
+      _hubConnectors[3],
+      abi.encodeWithSelector(IArbitrumHubConnector.processMessageFromRoot.selector),
+      abi.encode()
+    );
+    vm.prank(_autonolas);
+    proxy.processFromRootKeep3r(abi.encode(params), 421613, bytes32(uint256(4)));
+  }
+
+  function test_RelayerProxyHub__processFromRootKeep3r_worksForOptimism() public {
+    vm.mockCall(address(_keep3r), abi.encodeWithSelector(IKeep3rV2.isKeeper.selector, _autonolas), abi.encode(true));
+    vm.mockCall(
+      _hubConnectors[4],
+      abi.encodeWithSelector(IOptimismHubConnector.processMessageFromRoot.selector),
+      abi.encode()
+    );
+    vm.roll(100);
+    vm.prank(_autonolas);
+    Types.WithdrawalTransaction memory _tx = Types.WithdrawalTransaction({
+      nonce: 0,
+      sender: address(0),
+      target: address(1),
+      value: 10,
+      gasLimit: 100,
+      data: abi.encode("data")
+    });
+    Types.OutputRootProof memory _outputRootProof = Types.OutputRootProof({
+      version: bytes32(uint256(1)),
+      stateRoot: bytes32(uint256(2)),
+      messagePasserStorageRoot: bytes32(uint256(3)),
+      latestBlockhash: bytes32(uint256(4))
+    });
+    bytes[] memory _withdrawalProof = new bytes[](1);
+    _withdrawalProof[0] = abi.encode("withdrawalProof");
+    IOptimismHubConnector.OptimismRootMessageData memory params = IOptimismHubConnector.OptimismRootMessageData({
+      _tx: _tx,
+      _l2OutputIndex: 42,
+      _outputRootProof: _outputRootProof,
+      _withdrawalProof: _withdrawalProof
+    });
+    proxy.processFromRootKeep3r(abi.encode(params), 10, bytes32(uint256(1)));
+
+    vm.mockCall(
+      _hubConnectors[5],
+      abi.encodeWithSelector(IOptimismHubConnector.processMessageFromRoot.selector),
+      abi.encode()
+    );
+    vm.prank(_autonolas);
+    proxy.processFromRootKeep3r(abi.encode(params), 420, bytes32(uint256(1)));
+  }
+
+  function test_RelayerProxyHub__processFromRootKeep3r_worksForZkSync() public {
+    vm.mockCall(address(_keep3r), abi.encodeWithSelector(IKeep3rV2.isKeeper.selector, _autonolas), abi.encode(true));
+    vm.mockCall(
+      _hubConnectors[6],
+      abi.encodeWithSelector(IZkSyncHubConnector.processMessageFromRoot.selector),
+      abi.encode()
+    );
+    vm.roll(100);
+    vm.prank(_autonolas);
+    IZkSyncHubConnector.ZkSyncRootMessageData memory params = IZkSyncHubConnector.ZkSyncRootMessageData({
+      _l2BlockNumber: 42,
+      _l2MessageIndex: 69,
+      _l2TxNumberInBlock: 24,
+      _message: abi.encode("data"),
```

Comment:
"fuzz me" - `_message`

Reply:
But this test isn't fuzzed, nor does it seem like it needs to be.